boards/arm/adi_sdp_k1/arduino_r3_connector.dtsi

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics
```

Comment:
is this (c) correct?

Reply:
I copied the original file from another board. Just updated the copyright accordingly.